tests/configs/netcdf_harvester_config__valid.yaml

```diff
@@ -0,0 +1,31 @@
+file_meta:
+  cycle: '2015120206'
+  cycletime_str: '%Y%m%d%H'
+  filename_str: innov_stats.metric.%Y%m%d%H.nc
+  filepath: /home/slawrence/Development/rnr-score-harvester/tests/data
```

Comment:
feeling uneasy about this absolute path.

Reply:
Yeah, sorry about that, I'll remove it.

Reply:
This file is created during the unit test and shouldn't have been checked into the repo.